Jenkinsfile

```diff
@@ -64,6 +64,7 @@ pipeline {
     stage('Lint and Test') {
       environment {
         NODE_ENV = "development"
+        NODE_OPTIONS="--experimental-vm-modules"
```

Comment:
do we still need this for 18? 18 should have full module support
if not, do you want to goto 20 instead? I don't really like using the flag.

Reply:
this is only used during lint+test stage. I think it's more related to Jest implementation than Node version, see https://jestjs.io/docs/ecmascript-modules